Extensions/tweaks.js

```diff
@@ -303,6 +308,13 @@ XKit.extensions.tweaks = new Object({
 
 	run: function() {
 		this.running = true;
+		var wrap_tags_all_lines = XKit.extensions.tweaks.preferences.wrap_tags.value;
+		var wrap_tags_one_line = XKit.extensions.tweaks.preferences.wrap_tags_one_line.value;
+		if (wrap_tags_all_lines && XKit.interface.is_tumblr_page()) {
+			XKit.tools.add_css(".post_full .post_tags.fadeable { max-height: none; } .see-all-tags { display: none; }", "xkit_tweaks_collapsible_tag_display");
+		} else if (wrap_tags_one_line && XKit.interface.is_tumblr_page()) {
+			XKit.tools.add_css(".post_full .post_tags.fadeable { max-height: 21px; } .see-all-tags { display: block; }", "xkit_tweaks_collapsible_tag_display");
```

Comment:
This doesn't work if the `see-all-tags` thing isn't originally present on the post—i.e., if it has less then 4 lines of tags but more then 1. Instead, the further lines just get hidden with no indication or way to unhide them.